src/main/java/ua/danit/final_project/entities/BedLinenStats.java

```diff
@@ -1,6 +1,10 @@
 package ua.danit.final_project.entities;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'com.fasterxml.jackson.annotation.JsonIgnore'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=83&issues=AWVY-fZc2xJdnw6QkSxh&open=AWVY-fZc2xJdnw6QkSxh)

[ISSUE_KEY]: AWVY-fZc2xJdnw6QkSxh